torch/fx/passes/utils/fuser_utils.py

```diff
@@ -108,14 +108,16 @@ def fuse_as_graphmodule(gm: GraphModule,
     subgraph = Graph()
 
     node_to_placeholder: Dict[Node, Node] = {}  # mapping of nodes from old graph to placeholder in new graph
-    node_map: Dict[Node, Node] = {}       # mapping of nodes from old graph to new graph
+    node_map: Dict[Node, Node] = {}         # mapping of nodes from old graph to new graph
 
     # handles inputs throught graph.node_copy's arg_transform functions
     def remap_inputs(x):
-        if x.op == "get_attr":
-            # TODO: do we really need copy the get_attr node into the graph?
-            # do something here
-            pass
+        if x.op == "get_attr" and x not in node_map.keys():
+            # Copy get_attr nodes as get_attr nodes. lift_graph_as_submodule call below ensures that
+            # the data is copied efficiently. The get_attr node is left as-is in the original graph
+            # in case it is used elsewhere outside the submodule.
+            get_attr_node = subgraph.get_attr(x.name, type_expr=x.type)
```

Comment:
I think using the subgraph.copy function might be better since it'll also copy over any metadata attached to that node. Thoughts? @SherlockNoMad 
```
get_attr_node = subgraph.node_copy(x)
```

Reply:
Yes, node_copy is better here, as it will also copy the node.meta field.

Reply:
Thanks, I'll update with node_copy()